### ISSUE TITLE
controller not starting, configuration error

### ISSUE THREAD

**Issue by mimizone** [apache/incubator-openwhisk-deploy-kube#59]
Going through the installation process, I see that the controllers pods are not starting properly, complaining about a misconfiguration.

```
[2017-09-26T16:36:36.971Z] [ERROR] [??] [Config] required property controller.blackboxFraction still not set 
[2017-09-26T16:36:36.971Z] [ERROR] [??] [Controller] Bad configuration, cannot start. 
```

It impacts also nginx which appears to rely on the controller service.

All the configurations are the ones coming from the GitHub repo, nothing changed so far.

Deploying in Kubernetes 1.7.2 in an actual cluster deployed with Rancher (not Minikube nor local).

**Comment by dgrove-oss** [comment]
needs PR #56 merged in

**Comment by dgrove-oss** [comment]
actually, should have been fixed by #58 which was merged and included the fix in #56.

**Comment by mimizone** [comment]
thanks I was using the code from 24 hours ago. Everything starting properly now.
